### DISCUSSION TITLE
Can't submit the form

### DISCUSSION THREAD

**Discussion by D4rkglitch** [community/community#119707] [GitHub Education]
I have tried to upload the id proof multiple times, even waited for days but it doesn't work. When I press the proceed button after uploading the jpeg, the page just sort of refreshes, without any response, going back to the state as it was before uploading id proof.
Doesn't even provide a confirmation of submission.

**Comment by DanielaDoesCode** [comment]
Same thing is happening to me

**Comment by moozunch** [comment]
hi, have you tried with another device?
